galley/pkg/config/source/kube/apiserver/source.go

```diff
@@ -101,13 +101,13 @@ func (s *Source) Dispatch(h event.Handler) {
 // Start implements processor.Source
 func (s *Source) Start() {
 	s.mu.Lock()
-	defer s.mu.Unlock()
```

Comment:
@howardjohn This is the root cause,  race with `s.onCrdEvent`, which also try to acquire lock.